tools/startSubmodulesTRAVIS.sh

```diff
@@ -72,7 +72,7 @@ echo "3 - Verifying if last non merge commit is a subproject commit:"
 libModuleHash=$(git show "$commitHash" | grep "+Subproject commit" | cut -f3 -d" ")
 strLength=${#libModuleHash}
 echo "    Subproject commit hash: $libModuleHash"
-if [ \( -n "$libModuleHash" \) -a \( "$strLength" -eq 40 \) ]; then
+if [ \( -n "$libModuleHash" \) -a \( "$strLength" -gt 40 \) ]; then
```

Comment:
Were you getting submodule hashes longer than 40 characters?

Reply:
Here. Travis was getting the hashes from `lib/pkp` and `lib/ui-library`. And wasn't checking my branches correctly.

https://app.travis-ci.com/github/pkp/ojs/jobs/555742980#L331

Reply:
I think that's because the last commit was a merge commit... If you configure git with `rebase = true`, and use an interactive rebase to keep the submodule update commit up-to-date and the last commit in the list, you should be fine (https://docs.pkp.sfu.ca/dev/testing/en/continuous-integration#prepare-pull-requests-for-testing).

Reply:
I've used the `git add .`, `git commit -m "Submodule update ##henriqueramos/implementation_7171##"` for last commit before push.

But the submodule is listing the hashes for `lib/ui-library` and `lib/pkp`.

https://app.travis-ci.com/github/pkp/ojs/jobs/556298568#L332

Reply:
From the Travis job, click the commit link: 
![image](https://user-images.githubusercontent.com/200411/150443481-b183ce0c-cf44-43c8-96f5-3cd6fd9fe23c.png)
This will take you to the last commit:
![image](https://user-images.githubusercontent.com/200411/150443514-bd587fb1-1346-402f-b41a-1ffae8c3d9c5.png)
...which shows as a merge commit.

Reply:
For posterity: Travis generates these merge commits when working with pull requests. https://travis-ci.community/t/travis-commit-is-not-the-commit-initially-checked-out/3775